internal/engine/eval/trusty/trusty.go

```diff
@@ -296,135 +294,201 @@ type alternative struct {
+		PackageType:    packageType,
+		PackageVersion: &dep.Dep.Version,
+	}
+
+	respSummary, err := trustyClient.Summary(ctx, input)
+	if err != nil {
+		return nil, fmt.Errorf("failed getting summary: %w", err)
+	}
+
+	respPkg, err := trustyClient.PackageMetadata(ctx, input)
+	if err != nil {
+		return nil, fmt.Errorf("failed getting package metadata: %w", err)
+	}
+
+	respAlternatives, err := trustyClient.Alternatives(ctx, input)
 	if err != nil {
-		return nil, fmt.Errorf("failed to send request: %w", err)
+		return nil, fmt.Errorf("failed getting alternatives: %w", err)
 	}
 
-	res := makeTrustyReport(dep, resp)
+	respProvenance, err := trustyClient.Provenance(ctx, input)
+	if err != nil {
+		return nil, fmt.Errorf("failed getting provenance: %w", err)
+	}
+
+	res := makeTrustyReport(dep,
+		respSummary,
+		respPkg,
+		respAlternatives,
+		respProvenance,
+	)
 
 	return res, nil
 }
 
 func makeTrustyReport(
 	dep *pbinternal.PrDependencies_ContextualDependency,
-	resp *trustytypes.Reply,
+	respSummary *trustytypes.PackageSummaryAnnotation,
+	respPkg *trustytypes.TrustyPackageData,
+	respAlternatives *trustytypes.PackageAlternatives,
+	respProvenance *trustytypes.Provenance,
 ) *trustyReport {
 	res := &trustyReport{
-		PackageName:     dep.Dep.Name,
-		PackageVersion:  dep.Dep.Version,
-		PackageType:     dep.Dep.Ecosystem.AsString(),
-		TrustyURL:       makeTrustyURL(dep.Dep.Name, strings.ToLower(dep.Dep.Ecosystem.AsString())),
-		Score:           resp.Summary.Score,
-		IsDeprecated:    resp.PackageData.Deprecated,
-		IsArchived:      resp.PackageData.Archived,
-		ActivityScore:   getValueFromMap[float64](resp.Summary.Description, "activity"),
-		ProvenanceScore: getValueFromMap[float64](resp.Summary.Description, "provenance"),
-	}
-
-	res.ScoreComponents = makeScoreComponents(resp.Summary.Description)
-	res.Alternatives = makeAlternatives(dep.Dep.Ecosystem.AsString(), resp.Alternatives.Packages)
-
-	if getValueFromMap[bool](resp.Summary.Description, "malicious") {
-		res.Malicious = &malicious{
-			Summary: resp.PackageData.Malicious.Summary,
-			Details: preprocessDetails(resp.PackageData.Malicious.Details),
-		}
+		PackageName:    dep.Dep.Name,
+		PackageVersion: dep.Dep.Version,
+		PackageType:    dep.Dep.Ecosystem.AsString(),
+		TrustyURL:      makeTrustyURL(dep.Dep.Name, strings.ToLower(dep.Dep.Ecosystem.AsString())),
 	}
 
-	res.Provenance = makeProvenance(resp.Provenance)
+	addSummaryDetails(res, respSummary)
+	addMetadataDetails(res, respPkg)
+
+	res.ScoreComponents = makeScoreComponents(respSummary.Description)
+	res.Alternatives = makeAlternatives(dep.Dep.Ecosystem.AsString(), respAlternatives.Packages)
+
+	if respSummary.Description.Malicious {
+		res.Malicious = makeMaliciousDetails(respPkg.Malicious)
+	}
+
+	res.Provenance = makeProvenance(respProvenance)
 
 	return res
 }
 
-func makeScoreComponents(descr map[string]any) []scoreComponent {
+func addSummaryDetails(res *trustyReport, resp *trustytypes.PackageSummaryAnnotation) {
```

Comment:
Can we get a nil annotation response without an error?  If not, maybe pass through the concrete type or document that it can't be nil.